### PR TITLE
volume-controller: fix static check failures

### PR DESCRIPTION
**Pull request by sadlil**
**What type of PR is this?**
/kind cleanup
/kind bug

**What this PR does / why we need it**:
This PR fixes static check failures for the following packages
```
pkg/controller/volume/persistentvolume/testing	
pkg/controller/volume/scheduling
```
**Which issue(s) this PR fixes**:

Fixes #

Ref #81657 
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: pkg/controller/volume/scheduling/scheduler_binder_test.go [hunk 1]
```diff
@@ -269,18 +266,20 @@ func (env *testEnv) updateVolumes(t *testing.T, pvs []*v1.PersistentVolume, wait
 	}
 	if waitCache {
 		wait.Poll(100*time.Millisecond, 3*time.Second, func() (bool, error) {
-			for _, pv := range pvs {
```
Comment (msau42) [inline pkg/controller/volume/scheduling/scheduler_binder_test.go]: why is the check changed from checking all pvs to only checking the first?

Comment (sadlil) [reply pkg/controller/volume/scheduling/scheduler_binder_test.go]: Hi @msau42, actually the loop always returns after the first successful execution for pvs. That's why it changed to check only the first one to fix the static check error.

Comment (msau42) [reply pkg/controller/volume/scheduling/scheduler_binder_test.go]: I think that was a bug in the original test then.  It should be checking all pvs

**Review by fejta — COMMENTED** [review]
/approve

the hack change

File: pkg/controller/volume/scheduling/scheduler_binder_test.go (hunk 1 above)
Comment (sadlil) [reply pkg/controller/volume/scheduling/scheduler_binder_test.go]: sure, i will update it as so. thanks.

**Comment by sadlil** [comment]
/assign @msau42

I have changed the test to run on all the pvs.

**Comment by msau42** [comment]
/lgtm
/approve
/priority important-soon
Thanks!

**Review by fejta — COMMENTED** [review]
/approve

File: pkg/controller/volume/scheduling/scheduler_binder_test.go [hunk 2]
```diff
@@ -51,7 +51,6 @@ var (
 	preboundPVC         = makeTestPVC("prebound-pvc", "1G", "", pvcPrebound, "pv-node1a", "1", &waitClass)
 	preboundPVCNode1a   = makeTestPVC("unbound-pvc", "1G", "", pvcPrebound, "pv-node1a", "1", &waitClass)
 	boundPVC            = makeTestPVC("bound-pvc", "1G", "", pvcBound, "pv-bound", "1", &waitClass)
-	boundPVC2           = makeTestPVC("bound-pvc2", "1G", "", pvcBound, "pv-bound2", "1", &waitClass)
```
Comment (msau42) [inline pkg/controller/volume/scheduling/scheduler_binder_test.go]: this file should have already been covered by https://github.com/kubernetes/kubernetes/pull/81744

Comment (msau42) [reply pkg/controller/volume/scheduling/scheduler_binder_test.go]: oops wrong pr (got it reversed)

**Pull request merged**